linux-anvil-aarch64-cuda/Dockerfile

```
@@ -45,6 +45,14 @@ RUN if [ "$CENTOS_VER" -le "7" ]; then \
 # Remove preinclude system compilers
 RUN rpm -e --nodeps --verbose gcc gcc-c++
 
+# Fallback to CentOS vault for CentOS 8 support.
+RUN if [ "$CENTOS_VER" -eq "8" ]; then \
+        find /etc/yum.repos.d/ -name "CentOS-*.repo" -exec \
+             sed -i 's/mirrorlist/#mirrorlist/g;s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' {} \; && \
+        yum update -y && \
```

Needed to ensure CUDA packages are not updated unintentionally. This flag was used in the command below as well

```suggestion
        yum update -y --disablerepo=cuda && \
```